Client/src/App.jsx

```diff
@@ -47,6 +47,10 @@ function App() {
             path="/monitors/create"
             element={<ProtectedRoute Component={CreateMonitor} />}
           />
+          <Route
+            path="/monitors/clone/:monitorId"
+            element={<ProtectedRoute Component={CreateMonitor} />}
+          />
```

Comment:
I don't think there is a need for a new route here, I think we can simply add an optional path param to `create`

`path="monitors/create/:monitorId?`

and resuse the create path.